src/components/QrScan/QrScan.js

```diff
@@ -164,6 +137,7 @@ const QrScan = () => {
         setScannedData(results.data);
       },
       {
+        preferredCamera: 'environment',
```

Comment:
On iOS the scanner starts in facing mode even with the `preferredCamera` set to environment. I noticed that in line 163 if we set `video` to be ` facingMode: "environment" }` instead of `true`, the iPhone will start with the back camera instead.

Reply:
Thanks for the hint - latest commit uses the stream to set default, which works on my android device. Also should fix the click button twice issue.
I tested in ios simulator and it does not render properly - can you test on an ios device?

Reply:
On iOS cordova, the app renders and starts with the back camera, but I am still unable to switch the cameras - both are still recognized, though

Reply:
I can look into this more later but I think for iOS the issue is coming from `window.cordova.plugins.iosrtc.registerGlobals()` in cameraHelper.js - when I comment it out, I am able to switch cameras (although that creates other video-related issues)

Reply:
Sounds good - if the ios part is the only piece not working, we may want to merge this and roll the ios fix into the ios integration issue.

Reply:
That works for me! I've been working on an iOS integration branch today and those fixes might end up fixing this camera switching issue.